src/ci/azure-pipelines/steps/run.yml

```diff
@@ -198,3 +202,8 @@ steps:
   condition: variables['AWS_SECRET_ACCESS_KEY']
   continueOnError: true
   displayName: Upload CPU usage statistics
+
+# Log the date, even on failure. Attempting to debug SSL certificate errors.
+- bash: date
+  continueOnError: true
```

Comment:
Pretty sure this won't trigger: `continueOnError` tells Azure Pipelines to ignore this task failing, but `condition: succeeded()` is still implied. You should also add `condition: true`.

Reply:
My fix is https://github.com/rust-lang/rust/pull/63511.